src/cluster/src/check/mod.rs

```diff
@@ -379,16 +398,208 @@ impl ClusterChecker {
             Box::new(CreateServiceAccountPermission),
             Box::new(LoadBalancer),
```

Comment:
Load Balancer is check is not needed for local.   So we need to tie this to cluster type.

Reply:
I believe this was already being done before this PR. Can we move this to a future issue?